src/creature.cpp

```diff
@@ -715,17 +716,18 @@ void Creature::deal_projectile_attack( Creature *source, dealt_projectile_attack
     const int max_damage = proj.impact.total_damage();
     std::string message;
     game_message_type gmtSCTcolor = m_neutral;
+    double world_multiplier = get_option<int>( "MONSTER_RESILIENCE" ) / 100.0;
```

Comment:
It's 2 causing lgtm.com warnings on type casting. Changing this to `float` should resolve them.
```suggestion
    float world_multiplier = get_option<int>( "MONSTER_RESILIENCE" ) / 100.0;
```